classifiers/reference_quality/special_character_classifier/config.py

```diff
@@ -0,0 +1,44 @@
+from util.configs import build_classifier_function_config
+from util.enums import State, BricksVariableType, RefineryDataType, SelectionType
+from . import special_character_classifier, INPUT_EXAMPLE
+
+
+def get_config():
+    return build_classifier_function_config(
+        # strapi information
+        function=special_character_classifier,
+        input_example=INPUT_EXAMPLE,
+        issue_id=345,
+        tabler_icon="LanguageKatakana",
+        min_refinery_version="1.7.0",
+        state=State.PUBLIC.value,
+        type="python_function",
+        available_for=["refinery", "common"],
+        part_of_group=[
+            "reference_quality",
+        ],  # first entry should be parent directory
+        # bricks integrator information
+        cognition_init_mapping = {
+            "true": "Needs fix",
+            "false": "null"
```

Comment:
i think for cognition this wouldn't be a mapping but a varaible default since LABEL will be a variable. To still use the same structure we could set default values for variables for the cognition part with something like 
```suggestion
            "@@LABEL@@": "Needs fix"
```

Reply:
Added this!